aten/src/ATen/cpu/vec/vec256/vec256_bfloat16.h

```diff
@@ -26,7 +26,7 @@ static inline void cvtbf16_fp32(const __m128i& a, __m256& o) {
 }
 
 static inline void cvtbf16_fp32(const __m256i& a, __m256& o1, __m256& o2) {
-  __m128i lo = _mm256_extractf128_si256(a, 0);
+  __m128i lo = _mm256_castsi256_si128(a);
```

Comment:
why this change?

Reply:
This cast is an equivalent operation with zero-latency so it makes sense to use it instead. However the compiler almost certainly makes this change already so I can drop it if you think it's less clear.

Reply:
Yeah for people who don't have these instructions memorized the previous approach is easier to understand